packages/pluginmanager/src/widget.tsx

```diff
@@ -0,0 +1,307 @@
+              .map(data => {
+                return {
+                  data: data,
+                  key: data.id
+                };
+              })}
+            columns={[
+              {
+                id: 'plugin-id',
+                label: this.trans.__('Plugin'),
+                renderCell: (row: IEntry) => (
+                  <>
+                    <code>{row.id}</code>
+                    <br />
+                    {row.description}
+                  </>
+                ),
+                sort: (a: IEntry, b: IEntry) => a.id.localeCompare(b.id)
+              },
+              {
+                id: 'description',
+                label: this.trans.__('Description'),
+                renderCell: (row: IEntry) => <>{row.description}</>,
+                sort: (a: IEntry, b: IEntry) =>
+                  a.description.localeCompare(b.description),
+                isHidden: true
+              },
+              {
+                id: 'autostart',
+                label: this.trans.__('Autostart?'),
+                renderCell: (row: IEntry) => (
+                  <>
+                    {row.autoStart ? this.trans.__('Yes') : this.trans.__('No')}
+                  </>
+                ),
+                sort: (a: IEntry, b: IEntry) =>
+                  a.autoStart === b.autoStart ? 0 : a.autoStart ? -1 : 1
+              },
+              {
+                id: 'requires',
+                label: this.trans.__('Depends on'),
+                renderCell: (row: IEntry) => (
+                  <>{row.requires.map(v => v.name).join('\n')}</>
+                ),
+                sort: (a: IEntry, b: IEntry) =>
+                  (a.requires || []).length - (b.requires || []).length,
+                isHidden: true
+              },
+              {
+                id: 'extension',
+                label: this.trans.__('Extension'),
+                renderCell: (row: IEntry) => <>{row.extension}</>,
+                sort: (a: IEntry, b: IEntry) =>
+                  a.extension.localeCompare(b.extension)
+              },
+              {
+                id: 'provides',
+                label: this.trans.__('Provides'),
+                renderCell: (row: IEntry) => (
+                  <>
+                    {row.provides ? (
+                      <code title={row.provides.name}>{row.tokenLabel}</code>
+                    ) : (
+                      '-'
+                    )}
+                  </>
+                ),
+                sort: (a: IEntry, b: IEntry) =>
+                  (a.tokenLabel || '').localeCompare(b.tokenLabel || '')
+              },
+              {
+                id: 'enabled',
+                label: this.trans.__('Enabled'),
+                renderCell: (row: IEntry) => (
+                  <>
+                    <input
+                      type="checkbox"
+                      checked={row.enabled}
+                      disabled={row.locked || !this.model.isDisclaimed}
```

Comment:
Should we add a title informing that the top disclaimer must be acknowledge to enable the checkboxes?

Reply:
Done in b1ca664c3db82769fa7b95e96f946830cf663c25